### PR TITLE
v0.3.0 feed colors

### PR DESCRIPTION
**Pull request by ngokevin**
i plan on changing it we serialize the color by codename rather than a hard-coded hex. this allows us to not have to use inline styles for feed. Probably need to generate a styl file as well

### CODE REVIEW

File: dist/js/mozilla_languages.json [hunk 1]
```diff
@@ -455,6 +455,10 @@
     "native": "Bahasa Indonesia", 
     "English": "Indonesian"
   }, 
+  "sat": {
+    "native": "\u0938\u0902\u0924\u093e\u0932\u0940", 
+    "English": "Santali"
+  },
```
Comment (chuckharmston) [inline dist/js/mozilla_languages.json]: Probably needs a separate commit and version number.

**Comment by chuckharmston** [comment]
We should be generating a Stylus file in addition to JS, so we don't need to have the hex values stored in two places.

**Comment by chuckharmston** [comment]
Code looks good, though.

File: dist/js/mozilla_languages.json (hunk 1 above)
Comment (ngokevin) [inline dist/js/mozilla_languages.json]: this is just generated with ```make```. it's fine since it's not being used.

**Comment by ngokevin** [comment]
Keeping the JS file so Transonic can populate the color form.

**Comment by ngokevin** [comment]
pushing forward with this, i might need to make additional changes later as i see how this plays out.

**Pull request merged**